### PR TITLE
a few minor tweaks in the code and some fixes in rst syntax

### PR DESCRIPTION
**Pull request by fchapoton**
nothing really important, but basic house-cleaning

### CODE REVIEW

**Comment by fchapoton** [comment]
Ok. I have now undone these changes.

**Comment by fchapoton** [comment]
so this should be ready for merging, when you have time ; see also my other pull request

File: pySecDec/code_writer/make_package.py [hunk 1]
```diff
@@ -559,7 +561,7 @@ def _make_FORM_shifted_orders(positive_powers):
     r'''
     Write FORM code that defines the preprocessor
     variables
-    `shiftedRegulator`regulatorIndex'PowerOrder`shiftedOrderIndex''.
```
Comment (spj101) [inline pySecDec/code_writer/make_package.py:562]: This changes the variable name (so the comment is no longer correct).
Please remove the change to this line.

**Review by spj101 — CHANGES_REQUESTED** [review]
Need minor change to one comment, then we rebase and merge.

File: pySecDec/code_writer/make_package.py (hunk 1 above)
Comment (fchapoton) [reply pySecDec/code_writer/make_package.py]: ok, undone. But this line is not in correct rst syntax. You may like to fix it later.

**Review by spj101 — APPROVED** [review]

**Comment by spj101** [comment]
This pr passed our internal `Fast Check` runner. Happy to release it. 

Thank you for the contribution and sorry for our long delay here!

**Pull request merged**